build.rs

```diff
@@ -86,7 +88,12 @@ fn avro_{name}() {{
     }};
     let input: Value = serde_json::from_str(input_data).unwrap();
     let expected: Value = serde_json::from_str(expected_data).unwrap();
-    assert_eq!(expected, convert_avro(&input, context));
+    if expected.is_null() {{
+        // No expected data = no avro support
```

Comment:
How about we throw an exception or at least a warning when Avro transpiler is run with object metric path?
Although I filed a [ticket](https://mozilla-hub.atlassian.net/browse/DENG-2029) to get rid of Avro support, I don't know when this happens.

Reply:
Oh, I guess I can add another test case that catches this.
Because `avro` will already panic, because it doesn't know what to do with the JSON piece in the schema